app/controllers/api/v1/users/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Users
+      class OmniauthCallbacksController < Devise::OmniauthCallbacksController
+        respond_to :json
+        def google_oauth2
+          @user = User.from_omniauth(auth)
+
+          # TODO: Return JWT token on both success and error
+          if @user.persisted?
+            render_success
+          else
+            render_error
+          end
+        end
+
+        private
+
+        def auth
+          @auth ||= request.env['omniauth.auth']
+        end
+
+        def render_success
+          render json: { success: true }
```

Comment:
Just to be sure we're aligned, we will return a token in the next issue, right? 🙏 

#11